jdbc-adapter/virtualschema-jdbc-adapter/src/test/java/com/exasol/adapter/dialects/oracle/OracleSqlDialectIT.java

```diff
@@ -268,9 +274,9 @@ void testSelectNumber() throws SQLException {
             matchNextRowDecimal(result, "1234567890123456789012345678901234.6");
         }
         matchSingleRowExplain(String.format(query, EXA_NUMBER_TABLE_JDBC_NUMBER_TO_DECIMAL),
-                "SELECT CAST(\"A\" AS DECIMAL(36,1)) FROM \"" + ORA_NUMBER_TABLE + "\"");
+                "SELECT CAST(\"NUMBER_T\".\"A\" AS DECIMAL(36,1)) FROM \"" + ORA_NUMBER_TABLE + "\"");
```

Comment:
Use "ORA_NUMBER_TABLE" constant for consistency.

Same for other tables below.

Reply:
done